src/services/services.ts

```diff
@@ -5247,7 +5247,7 @@ namespace ts {
 
             // Try to get the smallest valid scope that we can limit our search to;
             // otherwise we'll need to search globally (i.e. include each file).
-            let scope = getSymbolScope(symbol);
+            let scope = reduceScopeIfPossible && getSymbolScope(symbol);

```

Comment:
i would just remove the handling for the private property case from getSymbolScope..